### DISCUSSION TITLE
mem0 doesnt trigger a memory event reliably

### DISCUSSION THREAD

**Discussion by amititash** [mem0ai/mem0#2086] [Q&A]
Hello, 

I am noticing that unless I have a user input that explicitly states what the response is about, mem0 doesnt trigger a memory event. 

So for instance

Assistant: whats your name
User: Barney

will not trigger a memory event even though the request does go the backend and can be see in the logs. 

Is this a bug or is there something in the docs I have missed. Note that in my implementation it is hard for me to explicitly tag something because I cannot choose the order in which questions will be asked and answered and when they will be asked and answered. 

Is there any good way of handling this scenario?

Thanks!

**Comment by Dev-Khant** [comment]
Hey @amititash, you might want to try our platform version for this scenario, as it can handle such cases.

**Reply by amititash** [reply]
Hi @Dev-Khant thanks for your response. I was trying the platform version only. I will check again. Best.

**Reply by Dev-Khant** [reply]
Sure, let me know.
